tests/wpt/web-platform-tests/fetch/nosniff/stylesheet.html

```diff
@@ -3,28 +3,36 @@
 <script src=/resources/testharnessreport.js></script>
 <div id=log></div>
 <script>
-  var fails = ["", "?type=", "?type=x", "?type=x/x"],
-      passes = ["?type=text/css", "?type=text/css;blah"]
+  var fails = [null, "", "x", "x/x"],
+      passes = ["text/css", "text/css;charset=utf-8"]
```

Comment:
Because hyper's mime parser turns `text/css;blah` into  (text, css,[]). This is to spec according to https://tools.ietf.org/html/rfc2045#section-5.1 which defines mime types parameters as `parameter := attribute "=" value`